src/core/tritonserver.cc

```diff
@@ -570,12 +570,10 @@ TRITONSERVER_LogMessage(
     case TRITONSERVER_LOG_VERBOSE:
       LOG_VERBOSE_FL(1, filename, line) << msg;
       break;
+    default:
+      LOG_ERROR_FL(__FILE__, __LINE__)
+          << "unknown logging level '" << level << "'";
   }
-
-  return TRITONSERVER_ErrorNew(
```

Comment:
Why dropping this?

Reply:
Dropped earlier because we were ignoring the error any way. The new allocation was not getting removed. Hence, logged the error in default above. I ahve incorporated David's suggestion above which takes care of the leak.